h/templates/client/annotation.html

```diff
@@ -154,18 +154,50 @@
     <div class="annotation-actions" ng-if="!vm.isSaving && !vm.editing() && vm.id()">
       <div ng-show="vm.isSaving">Saving…</div>
       <button class="small btn btn-clean"
-              ng-click="vm.reply()"
-              ><i class="h-icon-reply btn-icon"></i> Reply</button>
+        ng-click="vm.reply()">
+        <i class="h-icon-reply btn-icon"></i>
+        Reply
+      </button>
       <span class="share-dialog-wrapper">
         <button class="small btn btn-clean"
-                ng-click="vm.share($event)"
-                ><i class="h-icon-link btn-icon"></i> Link</button>
-        <span class="share-dialog" ng-click="$event.stopPropagation()">
+          ng-click="vm.share($event)">
+          <i class="h-icon-link btn-icon"></i>
+          Link
+        </button>
+        <span class="share-dialog share-dialog--actions" ng-click="$event.stopPropagation()">
           <a target="_blank"
-             class="h-icon-link"
-             ng-href="{{vm.annotationURI}}"
-             title="Open in new tab"></a>
-          <input type="text" value="{{vm.annotationURI}}" readonly>
+            class="h-icon-link"
+            ng-href="{{vm.annotationURI}}"
+            title="Open in new tab">
+          </a>
+          <input class="share-dialog__link" type="text" value="{{vm.annotationURI}}" readonly>
+        </span>
+      </span>
+      <span class="share-dialog-wrapper">
+        <button class="small btn btn-clean"
+          ng-click="vm.share($event)">
+          <i class="h-icon-share btn-icon"></i>
+        </button>
+        <span class="share-dialog" ng-click="$event.stopPropagation()">
```

Comment:
The new share dialog has a slight drop shadow in the design to make it stand out from the card a little. That needs to be added here - feel free to add it to both the new and old if that makes things easier.